ci-operator/config/openshift/rook/openshift-rook-master.yaml

```diff
@@ -0,0 +1,24 @@
+binary_build_commands: GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.15-openshift-4.6
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: GOPATH=$(go env GOPATH) make -j $(nproc) test
+tests:
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: GOPATH=$(go env GOPATH) make -j $(nproc) test
```

Comment:
why you need `GOPATH=$(go env GOPATH)` in every command?

Reply:
This is coming straight from our existing CI pipeline, perhaps it's not needed but I know nothing about this CI so I thought I'd just duplicate what we already do.